package.json

```diff
@@ -233,7 +233,7 @@
     "@vitejs/plugin-react": "^4.0.4",
     "ajv": "^8.12.0",
     "ajv-errors": "^3.0.0",
-    "arktype": "1.0.19-alpha",
```

Comment:
wow, we are using an `alpha`, that's not great, and now switching to `dev`. let's wait for the full release.

Reply:
thanks for the PR.

Reply:
@bluebill1049 The integration isn't very deep, I can guarantee these parts of the API will be the same when a stable 2.0 is released.

In the meantime, giving users access to a resolver that is compatible with the latest version seems like a big win. Any reason not to merge this now?

Reply:
@bluebill1049 Users have already asked to use 2.0 with `react-hook-form`. There will be stabilization period before there are no tags on the release that will not involve any of the APIs used in this PR.

Is there anything else I can do to get this merged?

Reply:
@bluebill1049 Sorry to both you on this, just looking to understand if there is anything I can do to move this along.